@here/harp-mapview/lib/MapView.ts

```diff
@@ -3059,7 +3059,7 @@ export class MapView extends THREE.EventDispatcher {
      * Gradually initialize & update TextRenderer as assets arrive.
      */
     private updateTextRenderer() {
-        if (this.m_theme.fontCatalogs === undefined) {
+        if (this.m_theme.fontCatalogs === undefined && this.m_theme.textStyles === undefined) {
```

Comment:
This reverts "fix" from #791 by requiring `textStyles` to be defined in order to render any text.
`textStyles` are by no means required in theme.

So, without further rationale i see this as invalid patch.

Reply:
So it means, a fontCatalogs must be defined in theme in order to render any labels?

Reply:
After offline discussion, it looks like bigger issue.
1) Theme change by `mapView.theme = ...` doesn't update existing instance of `TextElementsRenderer` and thus new values of `textStyles` and/or `fontCatalogs` are not obeyed

2) It's impossible to know when to initialize `TextElementsRenderer` before `Tile`s are loaded.

Thus, `TextElementsRenderer` should be initialized lazily only when first `TextElements`appear in MapView (either from `Tile`s or using APIs like `addOverlayText`).

Also, a followup change to update text config with new theme is needed.